docs/docs/identity-providers/readme.md

```diff
@@ -31,3 +31,173 @@ You must configure an IdP **[Service Account]** to write policy against group me
+            preferredUsernameAttr: uid
+            username: mail
+          usernamePrompt: Email
+        id: ldap
+        name: FreeIPA/LDAP
+        type: ldap
+      issuer: http://dex.YOURDOMAIN.dev
+      logger:
+        level: debug
+      oauth2:
+        responseTypes:
+        - code
+        skipApprovalScreen: false
+
+      staticClients:
+      # (Here I am creating static client for pomerium)
+      - id: pom
+        name: pom
+        redirectURIs:
+        # (pomerium authenticate service url)
+        - https://authenticate.YOURDOMAIN.dev/oauth2/callback
+        secret: pomerium
+
+```
+Below is configuration which supposed to be done in Pomerium
+
+```Note: I am using Pomerium helm chart```
+
+```
+config:
+  # routes under this wildcard domain are handled by pomerium
+  rootDomain: YOURDOMAIN.dev
+
+  policy:
+      # (give any name instead of hello, this will be the proxy url to access the particular service)
+    - from: https://hello.YOURDOMAIN.dev
+      # (give fqdn of the actual service which is being authenticated, here I am giving nextcloud service endpoint, which is running in nextcloud namespace)
+      to: http://nextcloud.nextcloud.svc.cluster.local:8080
+
+      # allowed_domains:
+          #(in general give here your domain)
+      #   - YOURDOMAIN.dev
+
+      # (If you want to give access to particular group members, I have tested this by creating devops group and members in that group, in freeipa)
+      allowed_groups:
+        - devops
+
+      # (If you want to give access to particular group members, I have tested this by creating devops group and members in that group, in freeipa)
+      allowed_idp_claims:
+        groups:
+        - devops
+
+  # (I didn't specify the root level CAs so)
+  insecure: true
+
+extraEnv:
+  # (This will give you details if user is not able to authenticate, ideally this should be turned off)
+  POMERIUM_DEBUG: true
+  LOG_LEVEL: "error"
+  IDP_SCOPES: "openid,profile,email,groups,offline_access"
+
+authenticate:
+  # (This we have set in dex's static client also remember! should be same)
+  redirectUrl: "https://authenticate.YOURDOMAIN.dev/oauth2/callback"
+
+  idp:
+    provider: oidc
+    clientID: pom
+    clientSecret: pomerium
+    # (your dex url)
+    url: http://dex.YOURDOMAIN.dev
+    scopes: "openid profile email groups offline_access"
+    # (for group based access policy)
+    serviceAccount: "pomerium-authenticate"
+
+ingress:
+  enabled: true
+  authenticate:
+    name: ""
+  secretName: ""
+  secret:
+    name: ""
+    cert: ""
+    key: ""
+  tls:
+    hosts: []
+  hosts: []
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/ingress.allow-http: "true"
+
+resources:
+  limits:
+    cpu: 150m
+    memory: 100Mi
+  requests:
+    cpu: 100m
+    memory: 100Mi
+
+```
+
+
+For more details and example you can visit [banzaicloud-example](https://banzaicloud.com/blog/pomerium-authentication/)
+
+Thanks
```

Comment:
Could you remove these two lines?

Reply:
Hey @calebdoxsey done, please have a look. btw thanks for the positive response:)